src/NzbDrone.Core/Indexers/NyaaTorrentRssParser.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System.Linq;
+using System.Xml.Linq;
+using NzbDrone.Common.Extensions;
+using NzbDrone.Core.Indexers.Exceptions;
+
+namespace NzbDrone.Core.Indexers
+{
+    public class NyaaTorrentRssParser : TorrentRssParser
+    {
+        public NyaaTorrentRssParser()
+        {
+        }
+
+        protected override long GetSize(XElement item)
+        {
+            var itemstripped = item.StripNameSpace();
+            if (itemstripped.Element("size") != null)
+            {
+                return ParseSize(itemstripped.Element("size").Value, true);
+            }
+
+            return base.GetSize(item);
+        }
+
+        protected override string GetInfoHash(XElement item)
+        {
+            var itemstripped = item.StripNameSpace();
+            if (itemstripped.Element("infoHash") != null)
+            {
+                return (itemstripped.Element("infoHash").Value);
+            }
+
+            return base.GetInfoHash(item);
+        }
+
+        protected override int? GetSeeders(XElement item)
+        {
+            var itemstripped = item.StripNameSpace();
+            if (itemstripped.Element("seeders") != null)
+            {
+                return int.Parse(itemstripped.Element("seeders").Value);
+            }
+
+            return base.GetSeeders(item);
+        }
+
+        protected override int? GetPeers(XElement item)
```

Comment:
Again similar to `InfoHashElementName` add one for peers and use that before parsing from description.